app/views/content/funding-and-support/if-youre-a-parent-or-carer.md

```diff
@@ -86,7 +90,7 @@ To be eligible, you need to:
 * have dependent children
 * have a household income of less than $finance_parentslearningallowance_income$ a year
 
-[Find out more about the Parents’ Learning Allowance and how to apply](https://www.gov.uk/parents-learning-allowance).
+$parents-learning-allowance$
```

Comment:
@emmaachesongray what's the reason behind adding a green arrow for one of the links but not the other two types of funding?